go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/cespare/cp v1.1.1 // indirect
 	github.com/codegangsta/negroni v1.0.0 // indirect
 	github.com/danielkov/gin-helmet v0.0.0-20171108135313-1387e224435e
+	github.com/davecgh/go-spew v1.1.1
```

Comment:
Where is this used?

Reply:
It's not. Removed in 17065d163e6a15cdc3b401a50dc8fddb284bfd6c.